iptc/ip6tc.py

```diff
@@ -273,7 +273,7 @@ def _create_mask(self, plen):
             if plen >= 8:
                 mask.append(0xff)
             elif plen > 0:
-                mask.append(2 ** plen - 1)
+                mask.append(0xff>>(8-plen)<<(8-plen))
```

Comment:
Oh right! This should've been `mask.append(0xff & (2 ** plen - 1))`. Your approach works too. 👍